src/app/src/components/FacilityListItemsTable.jsx

```diff
@@ -589,6 +589,14 @@ class FacilityListItemsTable extends Component {
                                     primary: '#00319D',
                                 },
                             })}
+                            menuPosition="fixed"
+                            menuPortalTarget={document.body}
+                            closeMenuOnScroll={e =>
+                                e.target.classList === undefined ||
+                                !e.target.classList.contains(
+                                    'css-4ljt47-MenuList',
```

Comment:
`'css-4ljt47-MenuList'`

I think it is a dynamically generated class name, isn't it?

Reply:
No. This class is always generated by react select for the drop-down items list.